postgres/datadog_checks/postgres/statements.py

```diff
@@ -296,6 +345,7 @@ def _load_pg_stat_statements(self):
                             pg_stat_statements_view=self._config.pg_stat_statements_view,
                             filters=filters,
                             extra_clauses="",
+                            called_queryids=', '.join([str(i) for i in self._query_calls_cache.called_queryids]),
```

Comment:
This is a little tricky, because the `_query_calls_cache` needs to get populated as well. Otherwise called queries can't get fetched on the next check run.

I'll push a commit for this and we can see how we think it looks.

Reply:
This is pushed now.